apps/todos/src/app/app.component.ts

```diff
@@ -1,17 +1,48 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { LoggingService } from './logging.service';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { NgForm } from '@angular/forms';
 
 @Component({
   selector: 'myproject-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss'],
+  styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'todos';
   toDisplay = false;
+  message = '';
+  count = 0;
+  userInput = 'Juli';
+  todoForm!: FormGroup;
+
+
+  constructor(private loggingService: LoggingService) {
+  }
+
+  ngOnInit() {
+    this.todoForm = new FormGroup({
+      todoName: new FormControl(''),
+      todoTime: new FormControl(''),
+      todoImportance: new FormControl('')
+    });
+  }
+
+  clicked() {
```

Comment:
Funktionen die bei Aktionen aufgerufen werden am Besten mit `on` - Prefixen.
z.B. :
```suggestion
  onMessageToggleButtonClicked() {
```